add dmail dapp

bin-dogami opened the pull request.

graemeblackwood commented on public/dapps/dmail/dmail-banner.png:
@bin-dogami The text will br cropped on smaller screen sizes. I recommend keeping anything important within the safe area defined in this Figma file: https://www.figma.com/community/file/1163928128813560560/dappland-content-template

graemeblackwood commented:
@bin-dogami Thanks for this. It looks like you're using an older version of the data JSON format. Updating `dotw` to `dotm` might fix it.

Also you list one contract, which is Verified (great to see!). Is this the only contract used by DMail?

bin-dogami commented:
> @bin-dogami Thanks for this. It looks like you're using an older version of the data JSON format. Updating `dotw` to `dotm` might fix it.
> 
> Also you list one contract, which is Verified (great to see!). Is this the only contract used by DMail?

hi,  I replaced the dotw` with `dotm`, is it right?
Also, I removed the contract, we have no contract yet.

graemeblackwood commented:
Thanks @bin-dogami – It looks like the app is live despite having no contracts. I am struggling to test it though because it wants a "Principal ID" for my recipient address (my other wallet).
